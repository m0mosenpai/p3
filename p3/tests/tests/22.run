../solution/wsh <tests/22.wsh
