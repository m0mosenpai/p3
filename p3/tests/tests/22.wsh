echo hello
uname
echo hello
history
