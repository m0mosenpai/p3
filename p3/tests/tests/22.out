wsh> hello
wsh> Linux
wsh> hello
wsh> 1) echo hello
2) uname
3) echo hello
wsh> 